package.json

```diff
@@ -77,28 +77,30 @@
     "url": "https://github.com/elastic/kibana.git"
   },
   "resolutions": {
-    "**/@types/node": ">=10.17.17 <10.20.0",
```

Comment:
Sorry for the messy diff. I sorted the list alphabetically because it was all over the place. The only change I made was adding these two lines:

```diff
+    "**/cross-fetch/node-fetch": "^2.6.1",
+    "**/isomorphic-fetch/node-fetch": "^2.6.1",
```